server/enterprise/metrics/metrics.go

```diff
@@ -1347,6 +1351,30 @@ func New(ps *platform.PlatformService, driver, dataSource string) *MetricsInterf
 	)
 	m.Registry.MustRegister(m.MobileClientSessionMetadataGauge)
 
+	m.DesktopClientCpuUsage = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: MetricsNamespace,
+			Subsystem: MetricsSubsystemClientsDesktopApp,
+			Name:      "cpu_usage",
+			Help:      "Average CPU usage of a specific process over an interval",
+			Buckets:   []float64{0, 1, 2, 3, 5, 8, 13, 21, 34, 55, 80, 100},
```

Comment:
The bucket size in the starting have a very fine duration, 1,2 where as in the end has much higher, is there a reason for this?

Reply:
We expect most values to be in the <5% range while the app is in use, anything over that is likely going to be a fair bit less cases, so I wanted to have enough buckets for that range.